target/linux/realtek/files-5.10/Documentation/devicetree/bindings/realtek,rtl8300-timer.yaml

```
@@ -0,0 +1,124 @@
+# SPDX-License-Identifier: GPL-2.0
+%YAML 1.2
+---
+$id: http://devicetree.org/schemas/timer/realtek,rtl8300-timer.yaml#
```

I think you can name your timer the 'otto' timer, just like we do the gpio and wdt drivers. The architecture is the otto architecture, and assuming they wrote their own VHDL for the gpio, wdt etc drivers, then this is the otto driver(s).

If they bought some IP, we should use that driver instead of course, but I somehow feel that that is less likely, we could compare registers with other drivers of course to see if there are close similarities.

```suggestion
$id: http://devicetree.org/schemas/timer/realtek,rtl-otto-timer.yaml#
```